recipes/plum-dispatch/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "plum-dispatch" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/plum-dispatch-{{ version }}.tar.gz
+  sha256: 766da66a93d0f00911777175dc7a43235ee3f18ead34c73977e710f732a918fd
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Are you using a compiler?
```suggestion
```
If not this can likely be noarch.

Reply:
yes, this is compiling a Cython module